.travis.yml

```diff
@@ -0,0 +1,32 @@
+language: node_js
+node_js:
+  - "8"
+
+os:
+  - linux
+
+git:
+  depth: 300
+
+deploy:
+  provider: pages
+  skip-cleanup: true
+  github-token: $GITHUB_TOKEN
+  keep-history: true
+  on:
+    branch: pull-monorepo-docs #for testing purposes
+  include:
+    - stage: copy v2,v3 readmes
+      script: /bin/bash ./update-readmes.sh
```

Comment:
I find it a bit confusing that we have both Travis CI and Jenkins configured for this repository. What is the interaction between these two CI servers? Could you please describe this setup in more details, e.g. in [README.md](https://github.com/strongloop/loopback.io/blob/gh-pages/README.md) or perhaps in a new file?

cc @rmg

Reply:
Good point. I will take a crack at documenting it.